src/apps/tools.clj

```diff
@@ -136,7 +144,7 @@
    (persistence/update-tool tool)
    (when container
      (set-tool-container id overwrite-public container))
-   (get-tool user id)))
+   (get-tool user id false)))
```

Comment:
I figured this and other 'update'/'set' type functions shouldn't show the defaults.

Reply:
That sounds reasonable to me too.